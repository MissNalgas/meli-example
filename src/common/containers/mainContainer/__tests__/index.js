import renderer from 'react-test-renderer';
import MainContainer from '..';

jest.mock('next/navigation', () => ({
	useRouter: () => ({})
}));

test('MainContainer renders correctly', async () => {

	const tree = renderer.create(<MainContainer>mainContainer</MainContainer>).toJSON();

	expect(tree).toMatchSnapshot();

});
