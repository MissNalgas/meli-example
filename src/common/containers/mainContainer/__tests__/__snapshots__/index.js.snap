// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer renders correctly 1`] = `
<div
  className="min-h-screen flex flex-col"
>
  <div
    className="bg-primary p-2"
  >
    <div
      className="flex gap-4 w-full max-w-screen-lg mx-auto"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        <img
          alt="ML logo"
          data-nimg="1"
          decoding="async"
          height={36}
          loading="lazy"
          onError={[Function]}
          onLoad={[Function]}
          src="/_next/image?url=%2FLogo_ML.png&w=128&q=75"
          srcSet="/_next/image?url=%2FLogo_ML.png&w=64&q=75 1x, /_next/image?url=%2FLogo_ML.png&w=128&q=75 2x"
          style={
            {
              "color": "transparent",
            }
          }
          width={53}
        />
      </a>
      <form
        className="rounded overflow-hidden flex bg-white flex-1"
        onSubmit={[Function]}
      >
        <input
          className="p-2 focus:outline-secondary h-10 w-full text-xgray"
          name="search"
          placeholder="Nunca dejes de buscar"
        />
        <button
          className="bg-light-gray h-10 w-10 grid place-content-center rounded"
          type="submit"
        >
          <img
            alt="Search icon"
            data-nimg="1"
            decoding="async"
            height={22}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fic_Search%402x.png&w=48&q=75"
            srcSet="/_next/image?url=%2Fic_Search%402x.png&w=32&q=75 1x, /_next/image?url=%2Fic_Search%402x.png&w=48&q=75 2x"
            style={
              {
                "color": "transparent",
              }
            }
            width={22}
          />
        </button>
      </form>
    </div>
  </div>
  <div
    className="bg-light-gray flex-1"
  >
    mainContainer
  </div>
</div>
`;
